.github/workflows/docs.yml

```diff
@@ -27,13 +27,14 @@ jobs:
       - id: gatherInfo
         # Collect the examples that need to be run, and the id of the latest
         # workflow run on the main branch.
+        permissions: read
```

Comment:
Is this required? I think `read` is the default anyway

Reply:
Hmm no read is not the default, because GITHUB_TOKEN until now has been used to deploy to gh-pages. In PRs that come from forks however GITHUB_TOKEN can only read, so the protection against attacks from PR submitters is already there by default, but not against github actions having some malicious code.

Reply:
Well, the code in a Github Action can do whatever it wants really, so if you give them the token they could in principle do whatever they want with your repo :sweat_smile: